rc/lsp.kak

```diff
@@ -105,6 +105,8 @@
 # Face for inlay hints.
 set-face global InlayHint cyan+d
 set-face global InlayCodeLens cyan+d
+# face for document links
+set-face global LspDocumentLink +u
```

Comment:
would be good to be consistent about the "Lsp" prefix (others like `InlayHint` don't have it).

I don't know much about faces but maybe we should prefer a generic name that can be used outside LSP.

Looking at the one precedent I can think of: doc.kak hardcodes the underline link
highlighting (`doc_render_ranges "%val{selection_desc}|default+u"`).

I'd be happy with hardcoding.

But if we want it configurable, I'd call it `Link`, because that's consistent with our `InfoLink`.